plugin/neovim-fuzzy.vim

```diff
@@ -191,10 +192,11 @@ function! s:fuzzy(choices, opts) abort
   " Clear the command line.
   echo
 
-  call writefile(a:choices, inputs)
-
-  let command = "fzy -l " . a:opts.lines . " > " . outputs . " < " . inputs
-  let opts = { 'outputs': outputs, 'handler': a:opts.handler, 'root': a:opts.root }
+  let fifo_name = tempname()
+  call system(['mkfifo', fifo_name])
+  let command = a:choices_cmd . " | tee " . fifo_name . " | fzy -l " . a:opts.lines . " > " . outputs
+  let num_choices_cmd = 'wc -l ' . fifo_name
+  let opts = { 'outputs': outputs, 'handler': a:opts.handler, 'cwd': a:opts.root }
```

Comment:
This is the second part of the fix - `root` means nothing to Neovim, but `cwd` actually makes it run the terminal in that directory.

I don't think it means anything to `:FuzzyOpen`, but if you ever want to make `:FuzzyGrep` run from the Git root this change is required.